### DISCUSSION TITLE
battery lifetime

### DISCUSSION THREAD

**Discussion by Masber** [nikitabobko/AeroSpace#647] [questions-and-answers]
dear aerospace community,

I am thinking in installing aerospace on my mac pro M3.
I have a question, does aerospace affect the battery life compare to original window manger?

thank you

**Comment by bcremer** [comment]
After a day of heavy use on my work machine the Activity Monitor reports no usage under "12 h Power". "Energy Impact" is at 0 most of the time.
Browsers or Slack are in the high hundreds. 

<img width="1587" alt="image" src="https://github.com/user-attachments/assets/93d783a2-27c4-4d4b-aa40-7083c8174f0b">
